src/types.ts

```diff
@@ -1,8 +1,6 @@
-import { Env } from './api/Env';
-
 export interface ConfigEntry {
```

Comment:
Isn't it what you need?
https://github.com/capsulajs/configuration-service/blob/develop/src/api/Entity.ts
I think we need to import it from here, if yes

Reply:
yes it may be, but I just updated what was there, I didn't want to make refactor, only removing unnecessary things